content/en/news/releases/1.6.x/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Upgrade Notes
+description: Important Changes to consider when upgrading to Istio 1.6.
+weight: 20
+---
+
+This page describes changes you need to be aware of when upgrading from Istio
+1.5.x to Istio 1.6.x. Here, we detail cases where we intentionally broke backwards
+compatibility. We also mention cases where backwards compatibility was preserved
+but new behavior was introduced that would be surprising to someone familiar with
+the use and operation of Istio 1.5.
+
+# Removal of Helm Installation
```

Comment:
This is super confusing as we still support helm3(?) and helm template(?).

Reply:
I think John's comment addressed this. Please let me know.